frontend/src/v5/ui/controls/circleButton/circleButton.styles.ts

```diff
@@ -15,83 +15,39 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import styled, { css } from 'styled-components';
-import { Fab } from '@mui/material';
-
-const SIZE_MAP = {
-	small: 22,
-	medium: 28,
-	large: 38,
-};
-
-const getButtonSize = (size) => {
-	const buttonSize = SIZE_MAP[size];
-
-	if (buttonSize) {
-		return css`
-			height: ${SIZE_MAP[size]}px;
-			width: ${SIZE_MAP[size]}px;
-			flex-shrink: 0;
-		`;
+import styled from 'styled-components';
+import { Button } from '@mui/material';
+
+export const BaseCircleButton = styled(Button)<{ disabled?: boolean }>`
```

Comment:
what happened with the size? It wasn't used anywhere?

Reply:
That's right. The only size used when I was tackling this issue was large, so there was no point in keeping it.

However, in custom tickets, I saw that a new variant of circleButton was created ("viewer") which used another size ("medium"). As such, I created a new component called ViewerButton to address the different size. In general, however, it feels nicer to override the css sizes rather than having a prop to control that, what do you think?